### DISCUSSION TITLE
Quando irão atualizar os Libs?

### DISCUSSION THREAD

**Discussion by Hakosk** [thatskymod/Sky-CotL-Scripts#143] [Q&A]
Somente os libs não estão funcionando, tentei entrar com eles desativados e funcionou. O Canvas está funcionando perfeitamente. 
Sabem me dizer quando eles vão atualizar os libs?

**Comment by thatskymod** [comment]
O problema se origina da biblioteca libLangit, que se tornou incompatível após a última atualização do jogo. Eu sinalizei a biblioteca como incompatível e lancei uma nova versão, [libLangit 0.4.4](https://github.com/thatskymod/Sky-CotL-Scripts/blob/05fa329f2e50de2d33b6232b5005705f45e5ac91/Canvas/Libs/libLangit044.so), que deve corrigir os travamentos imediatos.

<details>
  <summary>Show English version</summary>
  
  The issue originates from the libLangit library, which became incompatible after the latest game update. I have flagged the library as incompatible and released a new version, [libLangit 0.4.4](https://github.com/thatskymod/Sky-CotL-Scripts/blob/05fa329f2e50de2d33b6232b5005705f45e5ac91/Canvas/Libs/libLangit044.so), which should fix the immediate crashes.
  
</details>
